ledger/sandbox/src/main/scala/com/digitalasset/platform/apiserver/StandaloneApiServer.scala

```diff
@@ -91,6 +92,10 @@ final class StandaloneApiServer(
         "read" -> readService,
         "write" -> writeService,
       )
+      ledgerConfiguration = LedgerConfiguration(
+        initialConfiguration = initialConditions.config,
+        initialConfigurationSubmitDelay = Duration.ofSeconds(5),
```

Comment:
Could we make the submit delay configurable, e.g. via `ApiServerConfig`? In Canton it may take a variable amount of time until a participant connects to a domain.

Reply:
I have added an extra parameter to `StandaloneApiServer` of type `LedgerConfiguration`.

Reply:
Great thanks.

Not sure if I read the code correctly, but I couldn't find the default of 5 anymore - was assuming I'd find it in Kvutils Runner ledger config. But I don't know that code too well.